controllers/secretutils/secret-utils.go

```diff
@@ -0,0 +1,158 @@
+package secretutils
```

Comment:
license is missing and the package name should have been `controllers/util/secets.go`?